docs/standard/base-types/padding.md

```diff
@@ -28,7 +26,7 @@ ms.workload:
   - "dotnetcore"
 ---
 # Padding Strings in .NET
-Use one of the following <xref:System.String> methods to create a new string that consists of an original string that is padded with leading or trailing characters to a specified total length. The padding character can be spaces or a specified character, and consequently appears to be either right-aligned or left-aligned.  
+Use one of the following <xref:System.String> methods to create a new string that consists of an original string that is padded with leading or trailing characters to a specified total length. The padding character can be spaces or a specified character, and consequently appears to be either right-aligned or left-aligned. If the string's length is already equal to or greater than the desired total length, the padding methods return the existing string unchanged; for more information, see the **Returns** sections of the two overloads of the <xref:System.String.PadLeft%2A?displayProperty=nameWithType> and <xref:System.String.PadRight?displayProperty=nameWithType> methods.
```

Comment:
> The padding character can be spaces or a specified character, and consequently appears to be either right-aligned or left-aligned.

I find this sentence confusing, especially the usage of "consequently": how is the second part consequence of the first part?

I think I would write it like this:

> The padding character can be a space or a specified character, and the result can be either right-aligned or left-aligned.

Note that I also changed "spaces" to "a space".